scalafmt-tests/src/test/resources/default/Comment.stat

```diff
@@ -125,7 +125,7 @@ object a {
   /*
 
    /**
-     		* comment
```

Comment:
did you remove this because you create a separate test case for that?

Reply:
Exactly.   The case seemed to be multiplexed in a non-obvious (to me way).  
I de-multiplexed the concerns.

Creating a second test with a title which described its primary (and sole) concerned
seemed to be a gift to future people trying to suss this stuff.
The added complexity of a second test seemed worth the benefit.